examples/experimental/torch/classification/nas_advanced_main.py

```diff
@@ -0,0 +1,150 @@
+"""
```

Comment:
I would call it like `architecture_search_example.py` or even `bootstrap_nas.py`.

Reply:
Thanks! I've renamed to bootstrap_nas.py